web/org.eclipse.xtend.web/.classpath

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should perhaps be removed, and put on .gitignore? It has little value when commited, as it will be different for all users.. note e.g. how https://github.com/huibo-shi/xtext/tree/master/web/org.eclipse.xtext.web.example.jetty does not have a .classpath either (just a .project)